### PR TITLE
feat: render all projects in a MRA's get response

### PR DESCRIPTION
**Pull request by akshaykalia**
**Steps to test:**
- Run `shipyard get envs`/ `shipyard get env {uuid}` and ensure all the projects are listed in the table.

**Sample Response:**
```
                  UUID                 | READY |   APP NAME    |    REPO    | PR#  |                                URL
---------------------------------------|-------|---------------|------------|------|--------------------------------------------------------------------
  3879ecad-fe10-4452-8235-dd3b1ec1c256 | false | app-2         | app-2      |      |
  577f5b66-2966-4917-b9f1-260e3015752e | false | multi-app     | app-2      |      | https://multi-app.shipyardbuild.shipyard.host
                                       |       |               | shipyard   |      |
  f86ebeac-b8f7-46ba-96bd-acf6bc5388b0 | false | multi-app     | app-2      |      | https://multi-app-shipyard-pr1823.dev.shipyardbuild.shipyard.host
                                       |       |               | shipyard   | 1823 |
  9b936852-39f9-4131-9e20-86d36ea306cb | true  | shipyard      | shipyard   |      | https://shipyard.dev.shipyardbuild.shipyard.host
```

### CODE REVIEW

**Comment by akshaykalia** [comment]
Another option to represent the multi app

```
+--------------------------------+--------------------------------------+-------+---------------------+-----+-------------------------------------------------------------------------------------+
|            APP NAME            |                 UUID                 | READY |        REPO         | PR# |                                         URL                                         |
+--------------------------------+--------------------------------------+-------+---------------------+-----+-------------------------------------------------------------------------------------+
| now this is Podrac1ng          | 1d2355d2-8467-45ab-9eb8-55e269bbb129 | true  | httpbin             |     | https://1d2355d2-8467-45ab-9eb8-55e269bbb129.dev.qa.shipyard.pizza                  |
|                    16691663495 |                                      |       |                     |     |                                                                                     |
+                                +                                      +-------+---------------------+-----+                                                                                     +
|                                |                                      | true  | react-flask-starter |     |                                                                                     |
|                                |                                      |       |                     |     |                                                                                     |
+                                +--------------------------------------+-------+---------------------+-----+-------------------------------------------------------------------------------------+
|                                | efbe4d9e-f38e-475b-99fd-71c01e5e3d06 | false | httpbin             |     | https://None.dev.qa.shipyard.pizza                                                  |
|                                |                                      |       |                     |     |                                                                                     |
+                                +                                      +-------+---------------------+-----+                                                                                     +
|                                |                                      | false | react-flask-starter | 137 |                                                                                     |
|                                |                                      |       |                     |     |                                                                                     |
+--------------------------------+--------------------------------------+-------+---------------------+-----+-------------------------------------------------------------------------------------+
| react-flask-startersingle-repo | d9c84015-9aef-471d-b798-3e2748bc68fe | true  | react-flask-starter |     | https://d9c84015-9aef-471d-b798-3e2748bc68fe.dev.qa.shipyard.pizza                  |
| Application 16691663495        |                                      |       |                     |     |                                                                                     |
+                                +--------------------------------------+-------+---------------------+-----+-------------------------------------------------------------------------------------+
|                                | 01475939-21c6-46ed-b241-2036d36120f5 | false | react-flask-starter | 137 | https://shipyardtestorg-react-flask-sta-react-flask-sta-pr137.dev.qa.shipyard.pizza |
|                                |                                      |       |                     |     |                                                                                     |
+--------------------------------+--------------------------------------+-------+---------------------+-----+-------------------------------------------------------------------------------------+
```

File: cmd/env/get.go [hunk 1]
```diff
@@ -95,6 +95,42 @@ func NewGetAllEnvironmentsCmd() *cobra.Command {
 
 var ErrUnmarshalling = errors.New("failed to unmarshal environment(s)")
 
+// Converts the `environment` object to [][]string which is used during printing environments as table
+func convertObjToArray(env struct{ environment }) [][]string {
```
Comment (maxsokolovsky) [inline cmd/env/get.go:99]: This function can just take a value of type `environment`, no need to wrap it.
Also, the function needs a better name.

File: cmd/env/get.go [hunk 2]
```diff
@@ -95,6 +95,42 @@ func NewGetAllEnvironmentsCmd() *cobra.Command {
...
+	}
+
+	return data
+}
+
+// Render the environment data in tabular form
+func renderTable(data [][]string) {
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"App Name", "UUID", "Ready", "Repo", "PR#", "URL"})
+	table.SetAutoMergeCellsByColumnIndex([]int{0, 1, 5})
```
Comment (maxsokolovsky) [inline cmd/env/get.go:125]: Did you remove the borders to allow for separation of projects within one environment row?
I think it'd be nice to keep the minimal borders, if possible, so as to have less visual ASCII noise...

Comment (akshaykalia) [reply cmd/env/get.go:126]: Yeah, used the example for `SetAutoMergeCellsByColumnIndex` which had some noise.
Cleared it up a bit. Let me know what you think about this.

File: cmd/env/get.go (hunk 1 above)
Comment (akshaykalia) [reply cmd/env/get.go]: I was trying without `struct` but was having some issues. It was failing during the `make build`

Comment (maxsokolovsky) [reply cmd/env/get.go]: This is an error thrown by the Go compiler. You can fix it by changing the calls of the function by referencing the `environment` field of the `response.Data` (here it's a reference to the value whose name is the same as its embedded type - this is a less used feature of Go, took me a second to figure this out).

`data = append(data, extractDataForTableOutput(d.environment)...)` and
`data := extractDataForTableOutput(r.Data.environment)`

File: cmd/env/get.go (hunk 2 above)
Comment (maxsokolovsky) [reply cmd/env/get.go:126]: I like it! It's minimal and simple. We can always tweak those, of course. This, surprisingly, has a huge effect on the look of the output!

File: cmd/env/get.go (hunk 1 above)
Comment (akshaykalia) [reply cmd/env/get.go]: ah good find. pushed up the fix

**Pull request merged**